lambdas/models/pds_models.py

```diff
@@ -79,6 +91,13 @@ def get_current_home_address(self) -> Optional[Address]:
                 if entry.use.lower() == "home":
                     return entry
 
+    def get_ods_code_for_gp(self) -> str:
```

Comment:
I know it's cheeky to ask as I didn't write any when I first made this class, but maybe some unit tests for the pds models now the number of functions has increased just to verify everything works as expected?

Reply:
I've added a few tests